velox/substrait/SubstraitToVeloxPlanValidator.cpp

```diff
@@ -349,8 +384,8 @@ bool validateBoundType(::substrait::Expression_WindowFunction_Bound boundType) {
     case ::substrait::Expression_WindowFunction_Bound::kPreceding:
       break;
     default:
-      std::cout << "The Bound Type is not supported. "
-                << "\n";
+      VLOG(1)
```

Comment:
Should this part be added into log also?

Reply:
same as below.